lib/src/omni_datetime_picker.dart

```diff
@@ -18,6 +21,9 @@ class OmniDateTimePicker extends StatelessWidget {
       required this.type,
       this.selectableDayPredicate});
 
+  final bool? useDivider;
+  final String? title;
+  final TextStyle? titleStyle;
```

Comment:
Same as the previous comment, but would be also nice to add some docstring for the separator.

```suggestion
  /// A widget that separates the [title] - if not null - and the calendar, also separates between date and time pickers
  final Widget? separator;
  final Widget? title;
```